src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -26,6 +26,30 @@ export class TSConfigReader extends OptionsComponent
     private static OPTIONS_KEY:string = 'tsconfig';
 
 
+    private readConfigFile(event:DiscoverEvent, fileName:string) {
+        if (!FS.existsSync(fileName)) {
```

Comment:
I'm not very familiar with the project so here is how I've understood it so far:
1. [Here](https://github.com/TypeStrong/typedoc/blob/181a86d07a36170527ede0b6fbb870ade29d6086/src/lib/utils/options/readers/tsconfig.ts#L79) a typedoc intended only options may be present in the tsconfig.json, those options get merged with the raw `compilerOptions` from the tsconfig.json.
2. The conversion of the raw options to `CompilerOptions` and separation - TS/TypeDoc , is done [here](https://github.com/TypeStrong/typedoc/blob/181a86d07a36170527ede0b6fbb870ade29d6086/src/lib/utils/options/options.ts#L143) by [OptionDeclaration](https://github.com/TypeStrong/typedoc/blob/181a86d07a36170527ede0b6fbb870ade29d6086/src/lib/utils/options/declaration.ts#L76).

Why the conversion is done by typedoc and not TS I do not understand, I'm probably missing something or it is legacy. That's why I went with aggregating the raw options and not using the parsed ones returned from the `parseJsonConfigFileContent()` method used [here](https://github.com/TypeStrong/typedoc/blob/181a86d07a36170527ede0b6fbb870ade29d6086/src/lib/utils/options/readers/tsconfig.ts#L64) - it returns [ParsedCommandLine](https://github.com/Microsoft/TypeScript/blob/cf72ceb0933e0297ed9546cad3ae9def9916a114/src/compiler/types.ts#L3422) which has the parsed options - `options:CompilerOptions`.
If you have any suggestions please do tell.